src/lib/ai/tools/start-dm.ts

```
@@ -0,0 +1,82 @@
+import logger from '@/lib/logger';
+import { tool } from 'ai';
+import type { Message } from 'discord.js-selfbot-v13';
+import { z } from 'zod/v4';
```

ESLint

[error] Unable to resolve path to module 'zod/v4'.

(import-x/no-unresolved)